js/profile.js

```diff
@@ -0,0 +1,36 @@
+//dark mode button
+
+const bodyElement = document.querySelector('[data-js="body"]');
+const toggleBtn = document.querySelector('[data-js="button__dark-mode"]');
+let darkMode = localStorage.getItem("dark-mode");
```

Comment:
An welcher Stelle hast du die Klasse "dark-mode" definiert? Müsste ja irgendwo im HTML sein, oder?

Reply:
zu Edit2: Ja so habe ich das auch interpretiert. In den Beispiel (welches ich ja mehr oder weniger nur kopiert habe) war die Klasse "dark mode" auch nicht im HTML oder CSS. 
Daher vermute ich dass es ein Setting / Funktion des Browsers ist.